mathml/elements/mmultiscripts.json

```diff
@@ -43,6 +41,72 @@
             "standard_track": true,
             "deprecated": false
           }
+        },
+        "subscriptshift": {
+          "__compat": {
+            "support": {
+              "chrome": {
+                "version_added": false
+              },
+              "chrome_android": "mirror",
+              "edge": "mirror",
+              "firefox": {
+                "version_added": "1",
+                "version_removed": "83"
+              },
+              "firefox_android": "mirror",
+              "ie": {
+                "version_added": false
+              },
+              "oculus": "mirror",
+              "opera": "mirror",
+              "opera_android": "mirror",
+              "safari": {
+                "version_added": "9"
```

Comment:
Why is this 9 and all other safari versions are 7?

Reply:
I think it should be 7 too. Sorry...